firmware/Cargo.toml

```
@@ -89,11 +91,16 @@ keyberon = { git = "https://github.com/TeXitoi/keyberon", version = "0.2.0" }
 phf = { version = "0.11.2", default-features = false }
 packed_struct = { version = "0.10.1", default-features = false }
 rand = { version = "0.8.5", default-features = false, features = ["small_rng"] }
+rp-binary-info = { git = "https://github.com/rp-rs/rp-binary-info.git", commit = "141069dd1685885d35ce2eaa3b345b19288f56e4", optional = true}
```

```suggestion
rp-binary-info = { git = "https://github.com/rp-rs/rp-binary-info.git", optional = true}
```
Cargo doesn't use `commit` but `rev`, but I don't think we need to lock the commit explicitly